pipelines/smartseq2_multisample/README.md

```diff
@@ -4,22 +4,24 @@
 
 # Smart-seq2 Multi Sample Pipeline
 ## Introduction
-The Smart-seq2 Multi Sample pipeline is a wrapper around the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is developed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files and optionally generating a Loom file. 
+The Smart-seq2 Multi Sample pipeline is a wrapper around the [Smart-seq2 Single Sample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) pipeline. It is developed by the Data Coordination Platform of the Human Cell Atlas to process single-cell RNAseq (scRNAseq) data generated by Smart-seq2 assays. The workflow processes multiple cells by importing and running the [Smart-seq2 Single Sample workflow](/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) for each individual cell and then merging the resulting Zarr output files and optionally generating a Loom file. 
 
-Full details about the Smart-seq2 pipeline can be read in the [Smart-seq2 Single Sample workflow documentation](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_single_sample) in GitHub. 
+Full details about the Smart-seq2 pipeline can be read in the [Smart-seq2 Single Sample workflow](/pipelines/smartseq2_single_sample) documentation in GitHub. 
+
+The Multi Sample Smart-seq2 workflow can also be run in [Terra](https://app.terra.bio), a cloud-based analysis platform. The Terra [Smart-seq2 public workspace](https://app.terra.bio/#workspaces/featured-workspaces-hca/HCA%20Smart-seq2%20Multi%20Sample%20Pipeline) contains the Smart-seq2 workflow, workflow configurations, required reference data and other inputs, and example testing data.
```

Comment:
Regarding links into terra, is there a more permanent link that can be used here? If this pipeline is no longer featured at some point, will this link continue to work?

Reply:
This is the link. It's actually not a featured workspace (it's a public workspace), it's just under featured workspace billing account. Unless we change the name or funding, the workspace URL will remain the same.